newrelic/hooks/external_aiobotocore.py

```diff
@@ -44,5 +69,69 @@ async def wrap_endpoint_make_request(wrapped, instance, args, kwargs):
         return result
 
 
+async def wrap_client__make_api_call(wrapped, instance, args, kwargs):
+    # This instrumentation only applies to bedrock runtimes so exit if this method was hit through a different path
+    if not hasattr(instance, "_is_bedrock"):
+        return await wrapped(*args, **kwargs)
+
+    transaction = instance._nr_txn
+    if not transaction:
+        return await wrapped(*args, **kwargs)
+
+    # Grab all context data from botocore invoke_model instrumentation off the shared instance
+    trace_id = instance._nr_trace_id if hasattr(instance, "_nr_trace_id") else ""
+    span_id = instance._nr_span_id if hasattr(instance, "_nr_span_id") else ""
+    if hasattr(instance, "_nr_request_extractor"):
+        request_extractor = instance._nr_request_extractor
+    if hasattr(instance, "_nr_response_extractor"):
+        response_extractor = instance._nr_response_extractor
+    if hasattr(instance, "_nr_ft"):
+        ft = instance._nr_ft
```

Comment:
Can we change these to the 3 argument `getattr()` with a default of None? It'll still crash down below if one of these isn't hit.